Sources/TokamakDOM/Shapes/Path.swift

```diff
@@ -44,13 +44,14 @@ extension Path: ViewDeferredToRenderer {
       return AnyView(HTML("ellipse", ["cx": "50%", "cy": "50%", "rx": "50%", "ry": "50%"]
           .merging(stroke, uniquingKeysWith: uniqueKeys)))
     case let .roundedRect(roundedRect):
+      let ry = roundedRect.style == .continuous ? roundedRect.cornerSize.width : roundedRect.cornerSize.height
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakDOM/Shapes/Path.swift;
  Line: 47;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 110 characters (`line_length`)